recipe/migrations/singular4328.yaml

```diff
@@ -0,0 +1,7 @@
+__migrator:
+  build_number: 1
+  kind: version
+  migration_number: 1
+migrator_ts: 1701944250.6100492
+singular:
+- 4.3.2.8
```

Comment:
Oh, this version does not exist. It's 4.3.2.p8. I trusted that the bot would get this right…